std_msgs/Header header
string[] name
vision_msgs/BoundingBox2D[] boxes